libkbfs/conflict_resolver.go

```diff
@@ -3119,6 +3119,9 @@ func (cr *ConflictResolver) updateResolutionUsageAndPointers(
 	}
 	deletedBlocks := make(map[BlockPointer]bool)
 	for ptr := range toUnref {
+		if ptr == zeroPtr {
```

Comment:
This needs a comment explaining where these could come from, I think. Can they come from any of `originals`, `createdOriginals`, or `toUnrefPointers`, or can it only come from one or two?